vestido/vestido-storefront/modules/ProductListView/AddToCartButton.tsx

```diff
@@ -58,6 +92,7 @@ const AddToCartButton: React.FC<AddToCartButtonProps> = ({
   const buttonHeight = '40px';
   return (
     <>
+      <Toaster />
```

Comment:
Isn't this the Toaster outlet? Where the Toast gets shown? It's already defined at the root.